Dockerfile_with_testjob

```diff
@@ -0,0 +1,3 @@
+FROM nielsdenissen/flink-deployer:1.0.0
```

Comment:
`Dockerfile-including-sample-job` or `Dockerfile-including-test-job` as a name? You're still into Python ;)